openpgp/internal/ecc/ed448.go

```diff
@@ -53,6 +53,19 @@ func (c *ed448) UnmarshalByteSecret(s []byte) (d []byte) {
 	return s[1:]
 }
 
+func (c *ed448) UnmarshalSignature(r, s []byte) (sig []byte) {
+	// Only R is used, in prefixed native format draft-ietf-openpgp-crypto-refresh-06#section-9.2.1
```

Comment:
Increasingly nitpicky but this section provides a better explanation:
```suggestion
	// Only R is used, in prefixed native format, as per https://datatracker.ietf.org/doc/html/draft-ietf-openpgp-crypto-refresh-06#section-5.2.3.3.2
```